src/Symfony/Component/VarExporter/Tests/ProxyHelperTest.php

```diff
@@ -188,6 +188,28 @@ public function testCannotGenerateGhostForStringMagicGet()
         $this->expectException(LogicException::class);
         ProxyHelper::generateLazyGhost(new \ReflectionClass(StringMagicGetClass::class));
     }
+
+    /**
+     * @requires PHP 8.2
+     */
+    public function testNullStandaloneReturnType()
+    {
+        $classCode = <<<PHP
+        class TestNullStandaloneReturnType
+        {
+            public function foo(): null
+            {
+                return null;
+            }
+        }
+        PHP;
+        eval($classCode);
```

Comment:
this is the only way I've found to test this, otherwise, I get errors like `PHP Fatal error:  Null can not be used as a standalone type`

even with an anonymous class declared inside the test method

Reply:
Is that even the case if you move the definition of the `TestNullStandaloneReturnType` into its dedicated file?

Reply:
lol I was to biaised into using an inline class :facepalm: